src/components/editMeasure/populationCriteria/PopulationCriteriaHome.test.tsx

```diff
@@ -212,4 +231,101 @@ describe("PopulationCriteriaHome", () => {
     );
     expect(screen.getByTestId("groupDescriptionInput")).toHaveTextContent("");
   });
+
+  it("should not render Reporting component for non-QDM measures", () => {
+    render(
+      <MemoryRouter
+        initialEntries={[
+          { pathname: "/measures/testMeasureId/edit/reporting" },
+        ]}
+      >
+        <ApiContextProvider value={serviceConfig}>
+          <Route path={["/measures/testMeasureId/edit/reporting"]}>
+            <PopulationCriteriaHome />
+          </Route>
+        </ApiContextProvider>
+      </MemoryRouter>
+    );
+    // verifies if the side nav is created and Reportin is not available
+    expect(
+      screen.queryByRole("tab", {
+        name: /Reporting/i,
+      })
+    ).toBeNull();
+  });
+
+  it("should not render base configuration component for non-QDM measures", () => {
+    render(
+      <MemoryRouter
+        initialEntries={[
+          { pathname: "/measures/testMeasureId/edit/base-configuration" },
+        ]}
+      >
+        <ApiContextProvider value={serviceConfig}>
+          <Route path={["/measures/testMeasureId/edit/base-configuration"]}>
+            <PopulationCriteriaHome />
+          </Route>
+        </ApiContextProvider>
+      </MemoryRouter>
+    );
+    // verifies if the side nav is created and Reportin is not available
```

Comment:
Missing the `g` at the end of reporting. Should be "Base Configuration" in this case.

Reply:
Thank you, Will update it.